### ISSUE TITLE
Not working library?

### ISSUE THREAD

**Issue by 1nSaneRu** [Miljan9602/Instagram-API#1]
Hello! It's mgp25 fork? I install it via composer but code from "mgp25" not work. Missing Storage folder in Settings. When I upload this dir from mgp25 I get mismatch types...

This library work now? Mgp25 don't post more 28 days...

**Comment by Miljan9602** [comment]
@1nSaneRu Yes, it is mgp25 fork which is used for my own purposes and has a lot of changes. You can use it if you want to. It is actively maintained.

**Comment by 1nSaneRu** [comment]
I want to start using, but I can't do it because not all files in this branch. And I don't know all moments of changes. I'm change mgp25 branch to your using composer but can't start posting...

**Comment by jettero777** [comment]
@1nSaneRu привет, получилось запустить?
